scripts/Engineering/gui/engineering_diffraction/tabs/fitting/data_handling/data_model.py

```diff
@@ -149,30 +154,40 @@ def update_log_group_name(self):
 
     def get_ws_sorted_by_primary_log(self):
         ws_list = list(self._loaded_workspaces.keys())
+        tof_ws_inds = [ind for ind, ws in enumerate(ws_list) if
+                       ADS.retrieve(ws).getAxis(0).getUnit().caption() == 'Time-of-flight']
```

Comment:
instead of A`DS.retrieve(ws)` I think you can just use `self.loaded_workspaces[ws]` - I don't know if that's slightly quicker? both will work

Reply:
Good spot, probably will be quicker